website/content/v1.0/bare-metal-platforms/digital-rebar.md

```diff
@@ -5,10 +5,10 @@ description: "In this guide we will create an Kubernetes cluster with 1 worker n
 
 ## Prerequisites
 
-- 3 nodes (please see [hardware requirements](../../guides/getting-started#system-requirements))
+- 3 nodes (please see [hardware requirements](../../introduction/system-requirements/))
```

Comment:
i think we should use `rel` or `relref` so that hugo would validate that the links exist

Reply:
@frezbo fixed.